languages/csharp/exercises/concept/object-initializers/ObjectInitialization.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+
+//**** please do not modify the FacialFeatures class ****
```

Comment:
I was wondering if moving the `FacialFeatures` and `Identity` class declarations below the `Authenticator` class might result in the student understanding better which class to work on/less likely to modify these classes.

Reply:
Done.  Let's see what we think.  I assume we don't want this approach extended to enums and the like.  Perhaps just things that require a "do not modify" comment